Guide_for_Contributors_New_to_Github.md

```diff
@@ -0,0 +1,124 @@
+
```

Comment:
It's pretty common for content like this to live in a file called `CONTRIBUTING.md` - can we rename this file?

Reply:
Agree with this 💯

Reply:
yes, please feel free to change it! Perhaps `WIP CONTRIBUTING` while it is still in WIP?